Cargo.toml

```diff
@@ -18,16 +18,16 @@ name = "chrono"
 
 [features]
 # Don't forget to adjust `ALL_NON_EXCLUSIVE_FEATURES` in CI scripts when adding a feature or an optional dependency.
-default = ["clock", "std"]
-alloc = []
-std = ["alloc"]
+default = ["clock"]
```

Comment:
Not sure about this change? In our code, do we actually only need `std` for things that are related to `clock`? Or is this more of an incidental dependency rather than a conceptual one?

Reply:
We used to have custom code for getting the current time from the OS but today we are only a wrapper around `SystemTime::now()`.

Reply:
I guess what I mean is, is that the only use of `std` in the library? I guess all the other stuff is just `alloc`?

Reply:
We only use `std` for the `Error` trait, `SystemTime::now()`, and the stuff in the unix module.

Reply:
Okay, but our usage for the `Error` trait doesn't seem to be scoped to `clock` only, right?

Reply:
I've removed the commit.